common/src/java/org/apache/hive/http/HttpServer.java

```diff
@@ -702,6 +714,12 @@ private void initializeWebServer(final Builder b, int queueSize) throws IOExcept
       logCtx.setResourceBase(logDir);
       logCtx.setDisplayName("logs");
     }
+
```

Comment:
A comment would be nice; for each global filer, add a dedicated filter for each context handlers (if this is what the code actual intends :-))

Reply:
Added the comment